gdal/frmts/nitf/nitfimage.c

```diff
@@ -530,6 +530,20 @@ NITFImage *NITFImageAccess( NITFFile *psFile, int iSegment )
         }
     }
 
+    /* Expose a subset of the per-band metadata fields. This is similar in intent     */
+    /* to the disabled ESRI_BUILD SetBandMetadata() method. The ISUBCAT in particular */
```

Comment:
Esri told me they no longer use this ESRI_BUILD code paths. I'd suggest to re-enable SetBandMetadata(), remove its current content (which seems wrong) and repurpose it to expose ISUBCAT as a band metadata item.

Reply:
Thanks for the direction - I pushed up a couple commits that do just that.